delta/plugins/elasticsearch/src/main/resources/elasticsearch.conf

```diff
@@ -50,4 +50,7 @@ plugins.elasticsearch {
   listing-bucket-size = 500
   # set to false to disable Elasticsearch indexing
   indexing-enabled = ${app.defaults.indexing.enable}
+
+  # base to use to reconstruct resource identifiers in the proxy pass
+  proxy-id-base = "https://bbp.epfl.ch"
```

Comment:
What about putting this one in the fusion config ?

(Which can be renamed)

Also localhost could be the default value

Reply:
Made the changes. Also renamed `proxy-id-base` to `resolve-base`. If not suitable we can still change it later